OSOE-158: Evaluate using the Check Spelling action (#27)
Lombiq/GitHub-Actions: Docs/Workflows.md

      timeout-minutes: 10
```

> ## Check Spelling workflow

Reviewer: Similar to the other workflows, add an example of how to use this from another repo.

> Checks for spelling mistakes in a repository using the [Check Spelling](https://github.com/marketplace/actions/check-spelling) GitHub Action. There are 3 example files for filtering false positives:

Reviewer: Are these really "example" files?

> - `excludes.txt`: This file includes file names and extensions to be ignored

Reviewer: Put paths into `_this_`.

- `expect.txt`: This file contains plain text words that would be considered a spelling mistake.
- `patterns.txt`: This file contains patterns that would be considered a spelling mistake.

You should copy or create these files in the `.github/actions/spelling/` folder, 

> For more information visit the action's [wiki](https://github.com/check-spelling/check-spelling/wiki/Configuration#files).

Reviewer: ```suggestion
Checks for spelling mistakes in a repository using the [Check Spelling](https://github.com/marketplace/actions/check-spelling) GitHub Action. There are 3 example files for filtering false positives:
- `excludes.txt`: This file includes file names and extensions to be ignored.
- `expect.txt`: This file contains plain text words that would be considered a spelling mistake.
- `patterns.txt`: This file contains patterns that would be considered a spelling mistake.

You should copy or create these files in the `.github/actions/spelling/` folder, For more information visit the action's [wiki](https://github.com/check-spelling/check-spelling/wiki/Configuration#files).
```

      timeout-minutes: 10
```

## Spelling workflow

Checks for spelling mistakes in a repository using the [Check Spelling](https://github.com/marketplace/actions/check-spelling) GitHub Action. There are 3 configuration files for filtering false positives:
- _`excludes.txt`_: This file includes file names and extensions to be ignored.
- _`expect.txt`_: This file contains plain text words that would be considered a spelling mistake.

> - _`allow.txt`_: Same function as `expect.txt`.

Reviewer: Why do we have a second way of doing the same thing?

Author: Added explanation.

- _`patterns.txt`_: This file contains patterns that would be considered a spelling mistake.

There are more configuration files available, for more information visit the action's [wiki](https://github.com/check-spelling/check-spelling/wiki/Configuration#files).

> You can provide these files in your own repository, to the path `.github/actions/spelling`, this is restricted, and can not be configured for another path.

Reviewer: ```suggestion
You can provide these files in your own repository, under the path `.github/actions/spelling`. This can't be configured for another path.
```

## Check Spelling workflow

Checks for spelling mistakes in a repository using the [Check Spelling](https://github.com/marketplace/actions/check-spelling) GitHub Action. There are 3 configuration files for filtering false positives:
- _`excludes.txt`_: This file includes file names and extensions to be ignored.
- _`expect.txt`_: This file contains plain text words that would be considered a spelling mistake.
- _`patterns.txt`_: This file contains patterns that would be considered a spelling mistake.

There are more configuration files available, for more information visit the action's [wiki](https://github.com/check-spelling/check-spelling/wiki/Configuration#files).

You can provide these files in your own repository, the default path is `.github/actions/spelling`, but can be changed with the `config` parameter.

> You can also use already existing by setting the `spell-check-this` parameter to a repository.

Reviewer: "already existing"...?
